chapters/en/slides/module2/module2_01.md

```diff
@@ -0,0 +1,184 @@
+two slides to see which visual channels you find the most effective.
+
+---
+
+## Comparing sizes is easier for some geometrical objects than for others
+
+![](/module2/circle-comparison.png)
+
+Notes: In the image in this slide, try to estimate how many times larger
+the big circle is compared to the small one.
+
+---
+
+## Comparing sizes is easier for some geometrical objects than for others
+
+![](/module2/bar-comparison.png)
+
+Notes: In the image in this slide, try to estimate how many times larger
+the big bar is compared to the small one.
+
+---
+
+## Both the circles and rectangles differ by 7x
+
+<img src="/module2/7x.png" style="width:60.0%" />
+
+Notes: In both cases the answer is seven times bigger. Even if you
+guessed both of these exactly correct, most people find it is much
+easier to compare the length or position of the bars rather than the
+area of the circles. For the circles, you might even have hesitated at
+exactly what to compare when we said “how many time *larger*”, were you
+supposed to compare the area or the diameter? This is less ambiguous for
+bars as long as their widths are kept the same.
+
+This is important to keep in mind, especially when communicating to
+others via visualization, but also when creating plots for yourself.
+These two examples are originally from [Jeffrey Heer’s PyData
+talk](https://www.youtube.com/watch?v=hsfWtPH2kDg), who is visualization
+researcher at the university of Washington and whose research group
+created D3 and VegaLite (which Altair is based on).
+
+---
+
+## Summary of visual channel efficiency
+
+<img src="/module2/visual-channels-rank.png" style="width:80.0%" />
+
+Notes: Even if you got both these right yourself, the fact that many
+people prefer one over the other means that in order for you to create
+effective visualizations you need to know which visual channels are the
+easiest for humans to decode. Luckily, there has been plenty of research
+in this area, which can be summarizes in the schematic on this slide.
+
+Position is by far the best and therefore we should put our most
+important comparison there. Using position often means that we can’t use
+other things such as length or angle (like the angle in a pie chart),
+but we can add size or color to represent other relationships. Even if
+it is hard to tell exact information from these (is this color/dot 2x
+darker/bigger than another?) they are good to give and idea of trends in
+the data (are these dots darker/bigger than the others?
+
+---
+
+## Don’t use 3D without a good reason
+
+### Unnecessary 3D makes plot interpretation harder
+
+![](/module2/bad-3d-barchart.png)
+
+Notes: The biggest issues with using 3D is when it is used unnecessarily
+(like a 3D bar or pie chart), as the only way to compare position (like
+a 3D scatter plot), and when they are represented on a 2D medium like a
+paper where they can’t be rotated. In this slide we see a 3D bar chart,
```

Comment:
I think the bigger issue is not that they cannot be rotated, but that the extra cognitive load in reading them takes away from the pop out of differences.  The camera angle also is a thing, I agree.